xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -628,6 +628,7 @@ bool CVideoPlayerVideo::ProcessDecoderOutput(int &decoderState, double &frametim
         msg.player = VideoPlayer_VIDEO;
         msg.cachetime = DVD_MSEC_TO_TIME(50); // TODO
         msg.cachetotal = DVD_MSEC_TO_TIME(100); // TODO
+        msg.next_pts = this->m_messageQueue.GetTimeBack();
```

Comment:
this looks hacky and won't work for all cases like interlaced content or data based queue